### PR TITLE
Browser package

### PR DESCRIPTION
**Pull request by jonsequitur**
This introduces the basics for executing JavaScript when running .NET Interactive without a UI.

### CODE REVIEW

File: src/Microsoft.DotNet.Interactive.Browser/Microsoft.DotNet.Interactive.Browser.csproj [hunk 1]
```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```
Comment (brettfo) [inline src/Microsoft.DotNet.Interactive.Browser/Microsoft.DotNet.Interactive.Browser.csproj:4]: Why not `netstandard2.1`?

File: src/Microsoft.DotNet.Interactive.Browser/PlaywrightKernelConnector.cs [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+namespace Microsoft.DotNet.Interactive.Browser;
+
+public class PlaywrightKernelConnector : IKernelConnector
+{
+    public Task<Kernel> CreateKernelAsync(string kernelName)
+    {
+        var disposables = new CompositeDisposable();
+
+        var browserChannel = "msedge";
```
Comment (brettfo) [inline src/Microsoft.DotNet.Interactive.Browser/PlaywrightKernelConnector.cs:23]: What does this do on Mac/Linux?

File: src/microsoft-dotnet-interactive/lib/compositeKernel.d.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import * as contracts from "./contracts";
```
Comment (brettfo) [inline src/microsoft-dotnet-interactive/lib/compositeKernel.d.ts:1]: Rebase so this get's excluded.  Same for all other files in this directory.

**Review by brettfo — CHANGES_REQUESTED** [review]

File: src/Microsoft.DotNet.Interactive.Browser/PlaywrightKernelConnector.cs (hunk 2 above)
Comment (jonsequitur) [reply src/Microsoft.DotNet.Interactive.Browser/PlaywrightKernelConnector.cs:23]: We'll make this configurable later.

File: src/Microsoft.DotNet.Interactive.Browser/Microsoft.DotNet.Interactive.Browser.csproj (hunk 1 above)
Comment (jonsequitur) [reply src/Microsoft.DotNet.Interactive.Browser/Microsoft.DotNet.Interactive.Browser.csproj:4]: I'm not sure it's useful. The functionality here is never going to work in WASM, which is the primary reason we  support `netstandard2.1` in some packages.

**Review by colombod — CHANGES_REQUESTED** [review]

**Pull request merged**